sleap/info/write_tracking_h5.py

```diff
@@ -8,10 +7,15 @@
 
 The HDF5 file has these datasets:
 
-* "track_occupancy"     shape: tracks * frames
-* "tracks"              shape: frames * nodes * 2 * tracks
-* "track_names"         shape: tracks
-* "node_names"         shape: nodes
+* "track_occupancy"    (shape: tracks * frames)
+* "tracks"             (shape: frames * nodes * 2 * tracks)
+* "track_names"        (shape: tracks)
+* "node_names"         (shape: nodes)
+* "edge_names"         (shape: nodes - 1)
+* "edge_inds"          (shape: nodes - 1)
+* "point_scores"       (shape: frames * nodes * tracks)
+* "instance_scores"    (shape: frames * tracks)
+* "tracking_scores"    (shape: frames * tracks)
```

Comment:
Also include:

- `"labels_path"`: Path to the source .slp file (if available from GUI context)
- `"video_path"`: Path to source video (`video.backend.filename`)
- `"video_ind"`: Scalar integer index of the video within the source labels

Also nice to have:
- `"provenance"`: Copied from `labels.provenance`. Might be easier to convert to json string first.